app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -399,15 +406,14 @@ const setApplicationsDefaults = (application) => {
   return applicationWithDefaults
 }
 
-const mapProperties = ({ application, statusHistory, fileBaseUrl, units, availableUnits, rentalAssistances }) => {
```

Comment:
did you remove rentalAssistances here because you made it part of the `persistedApplication`?

Reply:
yes, `rentalAssistances` are now part of the suppl application form, so I wanted to make it consistent.